projects/wuffs/project.yaml

```diff
@@ -1,6 +1,8 @@
 homepage: "https://github.com/google/wuffs"
 language: c++
 primary_contact: "nigeltao@golang.org"
+auto_ccs :
```

Comment:
A very trivial note, feel free to ignore this if you like:
There seems to be an extra space between `auto_ccs` and `:`, making this line a bit inconsistent with the rest.